tracer/samples/AzureFunctionsWithAgentLessLogging/FunctionWithLogging/run.ps1

```diff
@@ -0,0 +1,133 @@
+## cf https://docs.datadoghq.com/tracing/other_telemetry/connect_logs_and_traces/dotnet/?tab=serilog
+Set-EnvironmentVariableForProcess -name "DD_LOGS_INJECTION" -value "true"
+Set-EnvironmentVariableForProcess -name "DD_ENV" -value "dev"
+Set-EnvironmentVariableForProcess -name "DD_SERVICE" -value "LogsInjectionInFunctionSamples"
+Set-EnvironmentVariableForProcess -name "DD_VERSION" -value "1.0.0"
+Set-EnvironmentVariableForProcess -name "DD_API_KEY" -value $ApiKey
+Set-EnvironmentVariableForProcess -name "DD_LOGS_DIRECT_SUBMISSION_INTEGRATIONS" -value "ILogger"
+
+if ($IsWindows) {
+    Write-Host "Using Windows configuration"
+    $AppDirectory = Get-ApplicationDirectory
+    $DotNetTracerHome = Join-Path $AppDirectory "datadog"
+
+    $ClrProfilerPath = ""
+    if ($Runtime -eq "x86") {
+        $ClrProfilerPath = Join-Path $DotNetTracerHome "win-x86\Datadog.Trace.ClrProfiler.Native.dll"
+    }
+    else {
+        $ClrProfilerPath = Join-Path $DotNetTracerHome "win-x64\Datadog.Trace.ClrProfiler.Native.dll"
+    }
+
+    if ($Framework -eq "net462") {
+        # Use the .NET Framework specific variables
+        Set-EnvironmentVariableForProcess -name "COR_ENABLE_PROFILING" -value "1"
+        Set-EnvironmentVariableForProcess -name "COR_PROFILER" -value "{846F5F1C-F9AE-4B07-969E-05C26BC060D8}"
+        Set-EnvironmentVariableForProcess -name "COR_PROFILER_PATH" -value $ClrProfilerPath
+    }
+    else {
+        # Use the .NET Core specific variables
+        Set-EnvironmentVariableForProcess -name "CORECLR_ENABLE_PROFILING" -value "1"
+        Set-EnvironmentVariableForProcess -name "CORECLR_PROFILER" -value "{846F5F1C-F9AE-4B07-969E-05C26BC060D8}"
+        Set-EnvironmentVariableForProcess -name "CORECLR_PROFILER_PATH" -value $ClrProfilerPath
+    }
+
+    # dotnet tracer home example: "..\Log4NetExample\bing\Debug\net7.0\datadog"
+    Set-EnvironmentVariableForProcess -name "DD_DOTNET_TRACER_HOME" -value $DotNetTracerHome
+
+    BuildAndRunSample
+}
+
+if ($IsLinux) {
+    Write-Host "Using Linux configuration"
+
+    $DotNetTracerHome = Join-PAth $AppDirectory "datadog"
+    $ClrProfilerPath = Join-Path $DotNetTracerHome "/linux-x64/Datadog.Trace.ClrProfiler.Native.so"
+    # if Alpine Linux x64
+    # $ClrProfilerPath = Join-Path $AppDirectory "/datadog/linux-musl-x64/Datadog.Trace.ClrProfiler.Native.so"
+    # if Linux ARM64
+    # $ClrProfilerPath = Join-Path $AppDirectory "/datadog/linux-arm64/Datadog.Trace.ClrProfiler.Native.so"
+
+    # Depending on 32-bit vs 64-bit the path to the profiler will change
+    Set-EnvironmentVariableForProcess -name "CORECLR_ENABLE_PROFILING" -value "1"
+    Set-EnvironmentVariableForProcess -name "CORECLR_PROFILER" -value "{846F5F1C-F9AE-4B07-969E-05C26BC060D8}"
+    Set-EnvironmentVariableForProcess -name "CORECLR_PROFILER_PATH" -value $ClrProfilerPath
+
+    Set-EnvironmentVariableForProcess -name "DD_DOTNET_TRACER_HOME" -value $DotNetTracerHome
+
+    BuildAndRunSample
+}
+
+if ($IsMacOs) {
```

Comment:
Nice, I must've missed this as I didn't know it was provided along with `IsWindows`/`IsLinux`